requirements_dev.txt

```diff
@@ -1,4 +1,6 @@
-requests
+crypt4gh~=1.7
```

Comment:
It is not obvious to me why this package was added to this PR, it doesn't seem related